### DISCUSSION TITLE
How to Change the Site/Domain Name Displayed in Google Search Results for My GitHub Pages Site

### DISCUSSION THREAD

**Discussion by AlexMisra1** [cotes2020/jekyll-theme-chirpy#1893] [Q&A]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

I'm using the **Chirpy Jekyll Theme** for my blog hosted on **GitHub Pages**. When I search for my site on Google, the search result displays "GitHub" as the site/domain name. However, I noticed that another site using the same theme and also hosted on GitHub Pages displays its title as the site/domain name in the search results instead of "GitHub". How can I customize the site/domain name in search results to show my site's title instead of "GitHub" while still having it hosted on GitHub Pages?

### Operations you have already tried

I haven't tried anything yet because I'm unsure where to start.

**Comment by kungfux** [comment]
Is there any chance that your GitHub profile or repository is showing up in Google search results instead of the blog, because I don't see any other explanation for why this is happening.

**Reply by AlexMisra1** [reply]
I just want to make clear what I'm asking.
You see, when searching up my name, this "GitHub" domain name (circled in red) appeared for my blog:
<img width="225" alt="Screenshot 2024-08-05 at 2 43 38 AM" src="https://github.com/user-attachments/assets/4fb131e7-d874-4e97-a7fc-768991183cce">
As well, another random guy with the same last name as me had his blog appear. I noticed that his blog has a GitHub Pages URL like mine, but his domain name (circled in red) is his name:
<img width="196" alt="Screenshot 2024-08-05 at 2 52 37 AM" src="https://github.com/user-attachments/assets/673c1a5e-242e-4a43-bc60-4d0cc50f135f">
I think it makes for for a cleaner look when my domain name is my name, so how do I make it like how he did it?

**Comment by kungfux** [comment]
Here is [documentation](https://developers.google.com/search/docs/appearance/site-names) about site names and it seems like Chirpy follows all the rules, so I'm not sure what else can be done. This may take a lot of time for experiments and the result will affect Google search results only while other search engines will display something different.

**Reply by kungfux** [reply]
It could be more complex solution as there are no strict published rules, but my guess was to add the `author` meta tag by specifying the author parameter in `_config.yml`. However, as I mentioned before, it takes a while to see if it helps.

```yml
author: Your name # the author name
```
